bedrock/base/templates/404-locale.html

```diff
@@ -0,0 +1,37 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol-mozilla.html" %}
+
+{% block page_title %}404 - Choose your language or locale to browse Mozilla.org{% endblock %}
+
+{% block page_desc %}
+  Select your country or region to indicate your preferred language.
+{% endblock %}
+
+{% block canonical_urls %}
+  {# Should 404s have canonical URLs? #}
```

Comment:
I think @a-kyne will know the best answer to that question, but I'd say maybe not, as it's not a page we generally drive traffic _to_